f2/src/BusinessPage.js

```diff
@@ -25,9 +25,7 @@ export const BusinessPage = () => {
         <Page>
           <Layout columns={{ base: 4 / 4, md: 6 / 8, lg: 7 / 12 }}>
             <Stack spacing={10} shouldWrapChildren>
-              <BackButton route="/whatwasaffected">
-                <Trans id="businessInfoPage.backButton" />
-              </BackButton>
+              <BackButton></BackButton>
```

Comment:
One more thing: Since `BackButton` doesn't have children anymore you can replace 
```
<BackButton></BackButton>
```
with
```
<BackButton />

Reply:
done